bigframes/core/indexes/base.py

```diff
@@ -618,6 +618,13 @@ def to_numpy(self, dtype=None, *, allow_large_results=None, **kwargs) -> np.ndar
     def __len__(self):
         return self.shape[0]
 
+    def item(self):
+        peeked = self.to_series().peek(2)
+        if len(peeked) == 1:
+            return peeked.iloc[0]
```

Comment:
Let's just call `peeked.item()` so that the error message matches pandas exactly.